src/microbe_stage/Meteor.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Collections.Generic;
+using ThriveScriptsShared;
+
+/// <summary>
+///   Contains definitions of meteors for meteor impact event
+/// </summary>
+public class Meteor : IRegistryType
+{
+    public WorldEffectVisuals VisualEffect;
+
+    public List<string> Chunks = new();
+
+    public Dictionary<Compound, double> Compounds = new();
+
+    public double Probability;
+
+    public string InternalName { get; set; } = null!;
+
+    public void Check(string name)
+    {
+    }
+
+    public void ApplyTranslations()
+    {
+        TranslationHelper.ApplyTranslations(this);
```

Comment:
This doesn't seem to have a name (just an internal name) so it looks to me like this does nothing?